### ISSUE TITLE
Update the README on DockerHub

### ISSUE THREAD

**Issue by siliconninja** [d8sychain/docker-mediawiki#9]
Hello,
Sorry if this is irrelevant, but I saw the instructions on DockerHub were out of date, because to use MariaDB, you have to run both a separate MariaDB container and a MediaWiki container, according to the instructions in the current `README.md`. The instructions on DockerHub used the `MYSQL_INSTALL_OPTION` environment variable.

There is a way to automatically update the README file on DockerHub [using the `sbt-native-packager` plugin](https://stackoverflow.com/a/34746930), but I haven't tested it yet. Alternatively, you could have the README on DockerHub be a link to point to the most recent README on the GitHub page.

Thank you!

**Comment by d8sychain** [comment]
Good catch, and still valid — thanks. Fixed the underlying staleness as part of the 1.35/1.39/1.43 modernization: README.md now accurately reflects the current supported tags, compose-based setup, and features (no more outdated `MYSQL_INSTALL_OPTION`-era instructions).

Docker Hub's description field itself has to be updated separately (Docker Hub doesn't auto-pull it from GitHub unless the repo is linked as an automated build source) — synced it manually for now. Closing this out; if it drifts again, linking the Docker Hub repo to GitHub under Manage Repository → Builds would keep it in sync automatically going forward.
